### PR TITLE
Issue #5510: fixes ImportOrder example for IntelliJ IDEA

### PR DESCRIPTION
**Pull request by jqno**
As discussed in #5510.

### CODE REVIEW

**Comment by jqno** [comment]
One check failed. Unfortunately I don't have a TeamCity account so I don't know how to fix this one.

**Comment by rnveach** [comment]
@jqno At the login screen, there is an option for a guest account.
It was just a simple timeout error, I will re-run it.

**Comment by jqno** [comment]
@rnveach I hadn't noticed the button. Thanks for re-running though!

File: src/xdocs/config_imports.xml [hunk 1]
```diff
@@ -1557,22 +1557,30 @@ import java.util.stream.IntStream;
           <li>groups are separated by, at least, one blank line and aren't separated internally</li>
         </ul>
 
-        <p>
-        Note: &quot;separated&quot; option is disabled because IDEA default has blank line
-        between &quot;java&quot; and static imports, and no blank line between
-        &quot;javax&quot; and &quot;java&quot;
-        </p>
+        <p>Note: a <a href="config_filters.html#SuppressionFilter">suppression filter</a>
+           is needed because IDEA has no blank line between "javax" and "java".</p>
```
Comment (romani) [inline src/xdocs/config_imports.xml]: please add  `ImportOrder has by design limitation to enforce empty line between groups ("java", "javax"), there is no flexibility to enforce empty lines between some groups and no empty lines between another groups.`

File: src/xdocs/config_imports.xml [hunk 2]
```diff
@@ -1557,22 +1557,30 @@ import java.util.stream.IntStream;
...
         </source>
 
+        <source>
+&lt;?xml version=&quot;1.0&quot;?&gt;
+&lt;!DOCTYPE suppressions PUBLIC
+    &quot;-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN&quot;
+    &quot;https://checkstyle.org/dtds/suppressions_1_2.dtd&quot;&gt;
+
+&lt;suppressions&gt;
+    &lt;suppress checks=&quot;ImportOrder&quot; message=&quot;^'java\..*'.*&quot;/&gt;
```
Comment (romani) [inline src/xdocs/config_imports.xml]: please add `<-- message contains no message text to works well in multi-language environment. -->

**Comment by jqno** [comment]
Thanks for the quick review! I've updated as requested.

**Comment by romani** [comment]
@jqno , please squash all in one commit and rebase latest master.

wrecker is green on latest master.

**Comment by romani** [comment]
@rnveach , please do final review

**Comment by rnveach** [comment]
wercker failure is unrelated

**Comment by romani** [comment]
@rnveach , please do not merge PRs with multiple commits. Author need to squash commits himself to not polite our git history.
You can do this on author behalf , please please use fast forward-merge with squash on your local.

**Comment by rnveach** [comment]
Sorry, I didn't realize there was multiple commits.

**Pull request merged**